### PR TITLE
updating the breadcrumbs markup to be WCAG compliant

### PR DESCRIPTION
**Pull request by travrob1**
Updating markup and visuals for breadcrumbs

Fix #104 

Test URLs:
- Before: https://main--petplace--hlxsites.hlx.page/article/cats/pet-care/cat-care/senior-cat-care/heres-older-cat-wants-attention-often
- After: https://issue-104--petplace--hlxsites.hlx.page/article/cats/pet-care/cat-care/senior-cat-care/heres-older-cat-wants-attention-often

### CODE REVIEW

File: scripts/scripts.js [hunk 1]
```diff
@@ -463,42 +463,52 @@ export function initializeTouch(block, slideWrapper) {
...
  */
 export async function createBreadCrumbs(crumbData) {
-  const breadcrumbContainer = document.createElement('div');
-  // Use the last item in the list's color
   const { color } = crumbData[crumbData.length - 1];
+  const breadcrumbContainer = document.createElement('nav');
+  breadcrumbContainer.setAttribute('aria-label', 'Breadcrumb');
 
+  const ol = document.createElement('ol');
+  ol.setAttribute('role', 'list');
```
Comment (ramboz) [inline scripts/scripts.js:471]: An `ol` already has `role="list"` behind the scene, so you don't need to add it.

File: scripts/scripts.js [hunk 2]
```diff
@@ -463,42 +463,52 @@ export function initializeTouch(block, slideWrapper) {
...
+  ol.setAttribute('aria-breadcrumb', 'true');
```
Comment (ramboz) [inline scripts/scripts.js:472]: I don't think this is a valid aria prop

File: scripts/scripts.js [hunk 3]
```diff
@@ -463,42 +463,52 @@ export function initializeTouch(block, slideWrapper) {
...
+
+  const homeLi = document.createElement('li');
+  homeLi.setAttribute('role', 'listitem');
```
Comment (ramboz) [inline scripts/scripts.js:475]: Same as `ol`, the `li` tag already implies `role="listitem"`, so yu can skip this

File: scripts/scripts.js [hunk 4]
```diff
@@ -463,42 +463,52 @@ export function initializeTouch(block, slideWrapper) {
...
   const homeLink = document.createElement('a');
-  homeLink.classList.add('home');
   homeLink.href = '/';
   homeLink.innerHTML = '<span class="icon icon-home"></span>';
   homeLink.setAttribute('aria-label', 'Go to our Homepage');
-  breadcrumbContainer.append(homeLink);
+  homeLink.setAttribute('aria-current', 'page');
```
Comment (ramboz) [inline scripts/scripts.js:480]: This property should only be set on the last link in the breadcrumb that actually represents the current page.
See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current#values for more details.

File: scripts/scripts.js [hunk 5]
```diff
@@ -463,42 +463,52 @@ export function initializeTouch(block, slideWrapper) {
...
+  homeLi.append(homeLink);
+  ol.append(homeLi);
 
   crumbData.forEach((crumb, i) => {
+    const li = document.createElement('li');
+    li.setAttribute('role', 'listitem');
```
Comment (ramboz) [inline scripts/scripts.js:486]: Can be skipped

**Comment by ramboz** [comment]
Might just wanna look at the CLS on the article page. Not sure if we just have flaky tests or maybe something that is a side-effect from the changes

File: scripts/scripts.js (hunk 2 above)
Comment (travrob1) [reply scripts/scripts.js:472]: Sure aint ;) Good catch.

**Pull request merged**